sql/size_utils.sql

```diff
@@ -328,9 +327,9 @@ BEGIN
                pg_namespace pns
                WHERE h.schema_name = %L
                      AND h.table_name = %L
-                     AND pgc.relname = h.table_name
+                     AND pgc.relname = c.table_name
```

Comment:
is this the actual fix?

Reply:
Yes that is the original fix. The second commit was an additional request by @davidkohn88 .